spec-main/api-net-spec.ts

```diff
@@ -1393,6 +1393,29 @@ describe('net module', () => {
       await collectStreamBody(nodeResponse);
       expect(nodeRequestProcessed).to.equal(true);
     });
+
+    it('should correctly throttle an incoming stream', async () => {
+      let numChunksSent = 0;
+      const serverUrl = await respondOnce.toSingleURL((request, response) => {
+        const data = randomString(kOneMegaByte);
+        const encoding = 'utf8';
```

Comment:
I think we can skip the encoding as it will be utf8 by default (and anyway we don't really care about the encoding for this test)
```suggestion
```